src/pages/docs/platform/auth/authentication.mdx

```diff
@@ -0,0 +1,1062 @@
+---
+title: Authentication Methods
```

Comment:
This is a huge page now, we have merged "basic and token" together. Leaving them separate is the best idea. When reseperationg, i suggest renaming "baisc auth" as "key auth" as thas what it is.